### DISCUSSION TITLE
iOS AR Quick Look loads GLB with no ios-src reference

### DISCUSSION THREAD

**Discussion by TheJackFace** [google/model-viewer#4668] [General]
Hi all, I've noticed that AR Quick Look in iOS Safari loads the GLB when no iOS source is referenced, is this documented somewhere? Is it actually displaying the GLB, or is the model somehow converted to USDZ on the fly?

**Comment by elalish** [comment]
Indeed, auto-generation of USDZ is one of our features, and it's documented here, among other places: https://modelviewer.dev/examples/augmentedreality/index.html#webXR

**Reply by GDEddie** [reply]
我可以调用它并生成二维码吗，然后在浏览器不支持的时候指引用户扫描二维码，类似这个这个页面https://nexreality.io/projects/watch/index.html的AR按钮
![Uploading Snipaste_2024-02-05_16-44-14.png…]()
